linera-storage/src/lib.rs

```diff
@@ -84,7 +84,7 @@ pub trait Storage: Sized {
     async fn contains_blob(&self, blob_id: BlobId) -> Result<bool, ViewError>;
 
     /// Lists the missing blobs from storage.
-    async fn missing_blobs(&self, blob_ids: Vec<BlobId>) -> Result<Vec<BlobId>, ViewError>;
+    async fn missing_blobs(&self, blob_ids: &[BlobId]) -> Result<Vec<BlobId>, ViewError>;
```

Comment:
To be in line with e.g. `contains_hashed_certificate_values`, should this return a `Vec<bool>` instead?

Reply:
I actually don't love returning a `Vec<bool>` for this, seems harder for the caller to check the result 🤔
`contains_hashed_certificate_values` isn't actually used anywhere, it seems, so I think I might just change it to also return the hashes of the missing hashed certificate values instead. What do you think?

Reply:
Ah, right, we were probably only using this for bytecode! Maybe we can just remove that one then?